poker/src/components/ConnectToLobby/ConnectToLobby.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { FC, useState, useEffect } from 'react';
+
+import c from './ConnectToLobby.module.scss';
+
+interface errorProps {
+    firstName: string;
+    lastName: string;
+    jobPosition: string;
+}
+
+const ConnectToLobby: FC = () => {
+  const [firstName, setFirstName] = useState<string>('');
+  const [lastName, setLastName] = useState<string>('');
+  const [jobPosition, setJobPosition] = useState<string>('');
```

Comment:
Maybe connect firstName, lastName, jobPosition in one useState
const handleChange = (event) => {
setSomeState((prevState) => {...prevState, [e.target.name]: e.target.value});

Reply:
fixed
2bea4a6